methods_test.test.js

```diff
@@ -0,0 +1,42 @@
+// import capitalize from './methods_to_test';
+const { capitalize, reverseString, analyse, Calculator, Cypher } = require('./methods_to_test.js')
+
+
+test("capitalize", () => {
+    expect(capitalize("name")).toMatch(/[N]ame/)
+})
+
+test("reverse string", () => {
+    expect(reverseString("string")).toMatch(/gnirts/)
+})
+
+test("calculator add", () => {
+    const calc = new Calculator
+    expect(calc.add(1,2,3)).toBe(6)
```

Comment:
This test is deceptive, why? It deceives us into believing our `add` function gives the correct answer. In fact, it gives you the wrong answer if we are to consider it should add the numbers.

```javascript
1 + 2 + 3 === 1 * 2 * 3 // will give you true
```
In short regardless of whether we use `add` or `multiply` to `1, 2, 3` as the three arguments it will always return 6. 

This is an evidence that we should write **MORE THAN ONE** test case for each function/unit we are testing. Let us write at least 3 cases.